### DISCUSSION TITLE
How to disable the add artwork feature when converting?

### DISCUSSION THREAD

**Discussion by s-fubuki** [enzo1982/freac#423] [Help & Support] [answered]
A function where the image in the current directory is added to the artwork.
Is there an option to disable this?

**Comment by DavidPesticcio** [comment, marked as answer]
For options that may help, have a look at the path/image below... :+1: 

Options | General Settings | fre:ac | Metadata | Cover art

![How to disable the add artwork feature when converting?](https://user-images.githubusercontent.com/8995466/213279251-4f48cb3a-5bda-4132-b687-ce515f1003c4.png)

**Reply by s-fubuki** [reply]
It is solved. Thank you very much.
